models/issue.go

```diff
@@ -1352,6 +1352,37 @@ type IssueStatsOptions struct {
 
 // GetIssueStats returns issue statistic information by given conditions.
 func GetIssueStats(opts *IssueStatsOptions) (*IssueStats, error) {
+	const maxIDListChunk = 200
```

Comment:
Maybe we can have a global value in `models` with the actual max number of parameters and calculate this from that value in case it needs to be used in more places.